iommu_registers.adoc

```diff
@@ -236,8 +239,11 @@ the IOMMU. At reset, the register shall contain the IOMMU supported features.
 |30    |`HPM`     |RO         | IOMMU implements a hardware performance monitor.
 |31    |`DBG`      |RO        | IOMMU supports the translation-request interface
 |37:32 |`PAS`      |RO        | Physical Address Size (value between 32 and 56)
-|47:38 | reserved  |RO        | Reserved for standard use
-|63:48 |_custom_   |RO        | _Reserved for custom use_
+|38    |`PD20`     |RO        | Three level PDT with 20-bit process_id supported.
```

Comment:
You may want to reorder here as well PD8, PD17 and PD20 to bits 38, 39 and 40

Reply:
Done.